index.js

```diff
@@ -20,18 +36,38 @@ var prefs = Cc["@mozilla.org/preferences-service;1"].getService(Ci.nsIPrefServic
 var Request = require("sdk/request").Request;
 var tabs = require("sdk/tabs");
 
+const DEBUG = true;
+const TAG = "bitcoin-price-ticker";
+const DATA_PROVIDERS_URL = "https://raw.githubusercontent.com/neoranga55/bitcoin-price-ticker/master/data/data-providers.json";
+const ADDON_UPDATE_DOCUMENT_URL = "http://neoranga55.github.io/bitcoin-price-ticker/";
```

Comment:
Line is too long.<br>'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).